src/lib.rs

```diff
@@ -602,12 +599,12 @@ pub mod pallet {
 	#[pallet::storage]
 	#[pallet::getter(fn init_relay_block)]
 	/// Relay block height at the initialization of the pallet
-	type InitRelayBlock<T: Config> = StorageValue<_, relay_chain::BlockNumber, ValueQuery>;
+	type InitVestingBlock<T: Config> = StorageValue<_, T::VestingBlockNumber, ValueQuery>;
```

Comment:
This implies a data migration right?

Reply:
We can use https://github.com/paritytech/substrate/pull/9625/files instead

Reply:
done in 2b348a0